specification/redis/resource-manager/Microsoft.Cache/stable/2022-05-01/redis.json

```diff
@@ -282,6 +293,12 @@
               "$ref": "#/definitions/RedisResource"
             }
           },
+          "202": {
+            "description": "The existing redis cache was successfully updated. Check Azure-AsyncOperation headers to poll detailed status.",
+            "schema": {
+              "$ref": "#/definitions/RedisResource"
+            }
+          },
```

Comment:
Is this PATCH long running operation?

Reply:
Yes, I pushed some updates to add `x-ms-long-running-operation` and addressed the warnings